handlers/whatsapp/whatsapp.go

```diff
@@ -88,10 +133,6 @@ func (h *handler) receiveEvent(ctx context.Context, channel courier.Channel, w h
 
 	// first deal with any received messages
 	for _, msg := range payload.Messages {
-		// ignore it if we aren't text
-		if msg.Type != "text" {
```

Comment:
Probably we keep this if block to continue if we have an type we do not support yet

Reply:
noted, will add the continue block below depending on where that bit of PR discussion goes.

Reply:
done